src/react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.3.6",
+  "version": "0.3.7",
```

Comment:
A note:  There's no need to bump this every time, just after a release.  The last release was `0.3.4` so we should be at `0.3.5` (with an implied prerelease)

Reply:
Whoops, that was me requesting this. Sorry!